### PR TITLE
FEF Handling added

### PR DESCRIPTION
**Pull request by SigsiuTrinity**
Adding handling for switching off the usage of FEF if another framework (e.g. Bootstrap) should be used.

### CODE REVIEW

File: component/backend/Model/Lifecycle.php [hunk 1]
```diff
@@ -39,7 +39,9 @@ public function onAfterBuildQuery(\JDatabaseQuery $query, $overrideLimits = fals
 			$lifecycleUserIDs = $this->getLifecycleUserIDs($when);
 			$db               = $this->container->db;
 
-			$query->where($db->qn('id') . ' IN (' . implode(',', $lifecycleUserIDs) . ')');
+			if ($lifecycleUserIDs) {
```
Comment (nikosdion) [inline component/backend/Model/Lifecycle.php]: This seems to be an unrelated commit. Please create a new branch in your repository with up to the previous commit. Then send a new PR from that branch.

Tip: please use a different branch for each PR you submit, not your master. Otherwise every commit you make into your master branch ends up being part of your PR.

Comment (SigsiuTrinity) [reply component/backend/Model/Lifecycle.php]: Thanks! Didn’t know. Was already wondering why it added the second one to the same PR. This is not what I wanted :)
Will try to correct it.

**Comment by nikosdion** [comment]
Closing this PR as I merged the correct one you filed. Thank you, Sigrid!
